targets/zephyr_targets/mqtts/src/config.h

```diff
@@ -0,0 +1,23 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#define MQTT_CLIENTID ""
+#define SNTP_SERVER "0.pool.ntp.org
+#define BROKER ""
+#define BROKER_PORT "8883"
+#define MQTT_BUFFER_SIZE 256u
+#define APP_BUFFER_SIZE 4096u
+#define MAX_RETRIES 10u
+#define BACKOFF_EXP_BASE_MS 1000u
+#define BACKOFF_EXP_MAX_MS 60000u
+#define BACKOFF_CONST_MS 5000u
+#define KEEP_ALIVE 60
+
+const char *mfThingId = " ";
+const char *mfThingKey = " ";
+const char *mfChannelId = " ";
+char mfTopic[150];
```

Comment:
Why 50? Magic number? Also, keep these numbers a power of two, 128 or 256 makes much more sense probably - unless this length is derived from MQTT standard, but then you need to explain this in the topic and reference exact standard chapter (provide the link as well). Use `DEFINE`.